ods_ci/tests/Tests/1000__model_serving/1009_model_serving_triton_on_keserve/1009_model_serving_triton_kserve.robot

```diff
@@ -0,0 +1,89 @@
+
+*** Test Cases ***
+
+Test PYTORCH Model Inference Via UI (Triton on Kserve)
+    [Documentation]    Test the deployment of an pytorch model in Kserve using Triton
+    [Tags]    Sanity    Tier1       RHOAIENG-9066
+
+    Open Data Science Projects Home Page
+    Create Data Science Project    title=${PRJ_TITLE}    description=${PRJ_DESCRIPTION}
+    ...    existing_project=${FALSE}
+    Open Dashboard Settings    settings_page=Serving runtimes
+    Upload Serving Runtime Template    runtime_filepath=${TRITON_RUNTIME_FILEPATH}
+    ...    serving_platform=single     runtime_protocol=REST
+    Serving Runtime Template Should Be Listed    displayed_name=${RUNTIME_NAME}
+    ...    serving_platform=single
+    Recreate S3 Data Connection    project_title=${PRJ_TITLE}    dc_name=model-serving-connection
+    ...            aws_access_key=${S3.AWS_ACCESS_KEY_ID}    aws_secret_access=${S3.AWS_SECRET_ACCESS_KEY}
+    ...            aws_bucket_name=ods-ci-s3
+    Deploy Kserve Model Via UI    model_name=${MODEL_NAME}    serving_runtime=kserve-tritonserver-pytorch-rest
+    ...    data_connection=model-serving-connection    path=triton/model_repository/    model_framework=pytorch - 1
+    Wait For Pods To Be Ready    label_selector=serving.kserve.io/inferenceservice=${MODEL_NAME}
+    ...    namespace=${PRJ_TITLE}
+    Run Keyword And Continue On Failure    Verify Model Inference With Retries
+    ...    ${MODEL_NAME}    ${INFERENCE_REST_INPUT_PYTORCH}    ${EXPECTED_INFERENCE_REST_OUTPUT_PYTORCH}    token_auth=${FALSE}
+    ...    project_title=${PRJ_TITLE}
+    [Teardown]    Run Keywords    Clean All Models Of Current User    AND
+    ...    Run Keyword If Test Failed    Get Kserve Events And Logs
+    ...    model_name=${MODEL_NAME}    project_title=${PRJ_TITLE}
```

Comment:
this must run before `Clean All Models Of Current User` otherwise it won't find any model pod for fetching the logs from